persist/src/main/java/com.codeforcommunity/database/seeder/Seeder.java

```diff
@@ -0,0 +1,62 @@
+package com.codeforcommunity.database.seeder;
+
+import com.codeforcommunity.database.records.CommentRecord;
+import com.codeforcommunity.database.records.PostRecord;
+import com.codeforcommunity.database.table.ICommentTable;
+import com.codeforcommunity.database.table.IPostTable;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Random;
+
+/**
+ * This class is what we'll be using to seed the database. When starting out, the database should be
+ * empty, so this will give you data to play around with. Later on, we'll use the data generated by
+ * this to test our program and make sure everything is working well.
+ */
+public class Seeder {
+  // A count of how many posts we want to make on startup for the in memory db. You can change this
```

Comment:
In the future, make these javadoc comments, `/** xxxx */`

Reply:
IntelliJ got mad when I did that. I think javadoc comments are only for outlining classes and methods instead of describing step by step what each call in a method does.

Reply:
I found [this](https://stackoverflow.com/questions/3607641/javadoc-comments-vs-block-comments) saying that javadoc should be used for a high level description of a class (info that the user of a class needs to know), while inline and block comments should be used for describing implementation details (info that a developer modifying a class needs to know)

Reply:
I agree that you should use inline comments for methods. I think you can use javadocs for fields of a class though. Is that true or is that not convention either?

Reply:
Ohhhh that's what you meant. Sorry, I thought you were talking about inlined comments in general. It looks like you can, so I'll work on that now.

Reply:
It's now done.